test/functional/mempool_persist.py

```diff
@@ -112,5 +112,17 @@ def run_test(self):
         assert_raises_rpc_error(-1, "Unable to dump mempool to disk", self.nodes[1].savemempool)
         os.remove(mempooldotnew1)
 
+        NUM_TXS_IN_MEMPOOL = 900
+        self.log.debug("Send %d transactions from node0 (to its own address)" % NUM_TXS_IN_MEMPOOL)
+        self.stop_nodes()
```

Comment:
A comment explaining why this is needed?

Reply:
I guess perhaps it is not needed...

Reply:
Please remove it, when it is not needed. Starting and stopping nodes is where a ton of time is wasted in the test framework.